lclibraries.lua

```diff
@@ -0,0 +1,420 @@
+if type(mesecon.luacontroller_libraries) == "table" then
```

Comment:
did we agree on anything last time we discussed the "luacontroller-library" thing?

IMO: this would be nice to have but won't ever get merged into the upstream mesecons, something like the `mooncontroller` would be a better fit (https://github.com/mt-mods/mooncontroller/blob/master/libraries.lua)
ideas, suggestions?